subsys/bluetooth/audio/bap_endpoint.h

```diff
@@ -27,6 +27,8 @@
 #define BROADCAST_STREAM_CNT 0
 #endif /* CONFIG_BT_BAP_BROADCAST_SOURCE */
 
+#define BAP_ENDPOINT_UNASSIGNED_ISO_ID 0xFF
```

Comment:
I get your point, but I don't think we can assign it like that. The Core specification does not specify `0xFF` to be `unassigned ISO`.
```
CIG_ID:
0x00 to 0xEF - Used to identify the CIG.
All other values - Reserved for future use

CIS_ID[i]:
0x00 to 0xEF - Used to identify a CIS
All other values - Reserved for future use
```
so `0xFF` is RFU value.

Reply:
Good point, I will make it so instead.

Reply:
I think that using the state as a requirement for setting these are good, however, it still doesn't solve uninitialized value. The user calling this function should now when this values have been set or not (which is why the RFU values was used initially).

How would you guys prefer that we solve this?
I could change the cis_id and cig_id to int type in the info struct, making negative values indicate that they have not been set. Otherwise some sort of flag could be used too, although I dont like to add extra entries.

Do you have any ideas or opinions?